src/http-client/fetch-client.ts

```diff
@@ -64,8 +69,88 @@ export class FetchClient implements HTTPClient {
     };
   }
 
+  /** {@inheritDoc HTTPStreamClient.stream} */
+  stream({
+    data,
+    headers: requestHeaders,
+    method,
+  }: HTTPStreamRequest): StreamAdapter {
+    const request = new Request(this.#streamURL, {
+      method,
+      headers: { ...requestHeaders, "Content-Type": "application/json" },
```

Comment:
That's why the driver does `body: JSON.stringify(data)`.

The API will also eventually support accepting a Query object as JSON, correct? It seems like it would be better to not have to flip the content type depending on the input.

Reply:
> That's why the driver does body: JSON.stringify(data).

The `stringify` is not called for the streaming endpoint. I'm sure you'd be betting a 400 if you were sending the token with quotes in it.

> The API will also eventually support accepting a Query object as JSON, correct? It seems like it would be better to not have to flip the content type depending on the input.

I'm not sure when / if that will came to be, honestly. In any case, when that happens, it'll require driver work to wire things up. We may switch to a more fleshed out JSON request body then.

Reply:
I'll set content type to text/plain for now.

Reply:
Body was updated to an object to handle start timestamps.

```typescript
export type StreamRequest = {
  token: string;
  start_ts?: number;
};
```